README.md

```diff
@@ -1,3 +1,25 @@
 ![](assets/party.gif)
 # Eggy-Bowl
 This is a Egg catcher Game. I designed and developed it. I do represent it with name "Eggy-Bowl". It is a 3-D graphical game using Multiple python libraries.
+
+This Game can be played using arrow keys.
+
+Left arrow key : to move catcher left
+
+Right arrow key : to move catcher right
+
+Built With
+### Python
+### tkinter
+### itertools
+### random
+
+Getting Started
+### To play download .exe and play.
+
+### To create your own download python and getstarted.
```

Comment:
The wording may be tweaked to be more direct and to have a stronger imperative tone. The second point is too vague and sounds a bit condescending... please reconsider it. A possible replacement would be along the lines of:

### Getting started
- **Software Requirements:** Python
- Clone the repo to your local device
- Run the .exe file

Reply:
Thanks for the suggestions @Akshu-on-github I will surely work on that...

Reply:
Python3*